src/polar_multigrid.cpp

```diff
@@ -586,23 +591,23 @@ void gmgpolar::prepare_op_levels()
                 v_level[l]->define_nz_P();
 
                 // // Number of nonzeros in the matrix
-                if (gyro::icntl[Param::verbose] > 2)
+                if (gyro::icntl[Param::verbose] > 3)
                     std::cout << "nz_P: " << v_level[l]->nz_P << "\n";
```

Comment:
doesnt this fit better into "print out all kinds of variable values" ? so gyro::icntl[Param::verbose]>5

Reply:
I don't think so. nz_P is not a variable but an information on the problem setting.